SharedContent/cpp-winui-template/WinUI3TemplateCpp/SettingsPage.xaml.cpp

```diff
@@ -0,0 +1,77 @@
+﻿#include "pch.h"
```

Comment:
@marb2000 @MikeHillberg the settings theme changes here aren't working currently. Would you be able to review this code?